{"uuid": "945fc0ce-08d9-4133-b645-cbd4366009cf", "children": ["b954171d-53f9-4244-842a-3c2886450c2f", "d9a4a0f4-250a-44d9-b95f-b6d8aa4b786b", "e2395fba-ed4e-4929-8dc7-2965eeb3ae12"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource\n\n", "trace": "  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/conftest.py\", line 36, in driver\n    request.cls.driver = webdriver.Remote(url, capabilities)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/appium/webdriver/webdriver.py\", line 274, in __init__\n    super().__init__(\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 266, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/appium/webdriver/webdriver.py\", line 369, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, parameters)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 418, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1646329655634, "stop": 1646329655639}], "start": 1646329655634, "stop": 1646329655777}